Orange/canvas/__main__.py

```diff
@@ -75,6 +75,58 @@ def fix_osx_10_9_private_font():
             pass
 
 
+def fix_macos_nswindow_tabbing():
+    """
+    Disable automatic NSWindow tabbing on macOS Sierra and higher.
+
+    See QTBUG-61707
+    """
+    import ctypes, ctypes.util, platform
+    if sys.platform != "darwin":
+        return
+    ver, _, _ = platform.mac_ver()
+    ver = tuple(map(int, ver.split(".")[:2]))
+    if ver < (10, 12):
+        return
+
+    c_char_p, c_void_p = ctypes.c_char_p, ctypes.c_void_p
+    id = Sel = Class = c_void_p
+
+    def annotate(func, restype, argtypes):
+        func.restype = restype
+        func.argtypes = argtypes
+        return func
+    try:
+        libobjc = ctypes.cdll.LoadLibrary(ctypes.util.find_library("libobjc"))
+        # For NSWindow class
+        AppKit = ctypes.cdll.LoadLibrary(ctypes.util.find_library("AppKit"))
```

Comment:
Is this import necessary? If not, remove. If it is, add a comment. Also, either remove variable `AppKit` or silence pylint warning.